rf-notebook/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM s22s/pyspark-notebook:spark-2.3.3-hadoop-2.7
```

Comment:
Interesting that we have had to fork the source and container repo, they don't provide a tag for this combo of versions?

Reply:
No. They only do the latest.